src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/output/PricesSpreadsheetGeneratorTest.kt

```diff
@@ -28,6 +28,7 @@ internal class PricesSpreadsheetGeneratorTest(@Autowired @Qualifier(value = "spr
     internal fun `standards moves are called for Serco`() {
         whenever(calculator.standardPrices(any())).thenReturn(sequenceOf())
         whenever(calculator.redirectionPrices(any())).thenReturn(sequenceOf())
+        whenever(calculator.longHaulPrices(any())).thenReturn(sequenceOf())
```

Comment:
Just curious why lines 30 and 31 are needed if you are only testing standard prices.  You are not verifying they are (not) called for example,

Reply:
I was otherwise getting a null pointer exception because the spreadsheet generator gets a list of prices to add for redirections and long hauls